### ISSUE TITLE
Warning: Accessing non-existent property 'cat' of module exports inside circular dependency

### ISSUE THREAD

**Issue by pschoffer** [lando/lando#2945]
tl;dr: Could you bump your `shelljs` dependency for the next release please?

**Tell us about your setup**

Lando v.3.0.26

Node.js v14.16.0

**Tell us about your `.lando.yml`**

Not relevant probably

**Tell us about the command you were running**

We are using lando as a library and trying to use it in our CLI directly. Our code is:

```
	const lando = new Lando( getLandoConfig() );
	await lando.bootstrap();

	const app = lando.getApp( instancePath );
	await app.init();

	await app.start();
```

**Tell us about the error you got**

```
node:233426) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:233426) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:233426) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:233426) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
...
```

If I trace warnings:
```
$ node --trace-warnings script.js
(node:238101) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:650:11)
    at Object.get (internal/modules/cjs/loader.js:664:5)
    at Object._register [as register] (/home/pavel/git/automattic/vip/node_modules/shelljs/src/common.js:455:12)
    at Object.<anonymous> (/home/pavel/git/automattic/vip/node_modules/shelljs/src/cat.js:4:8)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
....

```

**Tell us more**

So it seems the warning is coming from `shelljs`. It seems the folks there already [fixed it](https://github.com/shelljs/shelljs/pull/973). 

Could you bump your `shelljs` dependency for the next release, please?

**Comment by pirog** [comment]
@pschoffer please see https://github.com/lando/lando/issues/2928

**Comment by pschoffer** [comment]
Oh alright! Thanks for the link @pirog !
